utils/getDataTeamsAndScores.ts

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
remove empty line

Reply:
I added comments so that if people work with it will understand what is happening in my getDataTeamsAndScores util/ (helper) function